qchem/CHANGELOG.md

```diff
@@ -19,6 +19,30 @@ This release contains contributions from (in alphabetical order):
 
 <h3>New features since last release</h3>
 
+* The new function ``excitations_to_wires`` has been implemented to map the particle-hole
```

Comment:
@agran2018: now that v0.10 has been released, you will need to move this entry to the top section of the changelog :)

Reply:
Done, thanks @josh146 !.